torch/testing/_internal/inputgen/variable/type.py

```diff
@@ -0,0 +1,70 @@
+import math
+from enum import Enum
+
+import torch
+
+
+class ScalarDtype(Enum):
+    bool = bool
+    int = int
+    float = float
+
+
+SUPPORTED_TENSOR_DTYPES = [
+    torch.bool,
+    torch.uint8,
+    torch.int8,
+    torch.int16,
+    torch.int32,
+    torch.int64,
+    # torch.float16,
+    torch.float32,
+    torch.float64,
+    # torch.complex32,
+    # torch.complex64,
+    # torch.complex128,
+    # torch.bfloat16,
```

Comment:
why commented out?

Reply:
Because I want to support these dtypes in the short term. But I haven't verified my specifications with them yet.